src/rgw/rgw_sal_rados.cc

```diff
@@ -1420,6 +1427,18 @@ void RadosStore::finalize(void)
     rados->finalize();
 }
 
+void RadosStore::register_admin_apis(RGWRESTMgr_Admin* mgr)
+{
+  mgr->register_resource("bucket", new RGWRESTMgr_Bucket);
+  /*Registering resource for /admin/metadata */
+  mgr->register_resource("metadata", new RGWRESTMgr_Metadata);
+  mgr->register_resource("log", new RGWRESTMgr_Log);
```

Comment:
these will be required for multisite support in all stores

Reply:
We can move things around once there's multiple implementations.  At the moment, building modules is my goal.